dashboards-observability/public/components/application_analytics/helpers/utils.tsx

```diff
@@ -5,5 +5,12 @@
 
 // Name validation 0>Name<=50
 export const isNameValid = (name: string) => {
```

Comment:
question: do we need to handle cases like name is just a tab or space(s)?

Reply:
That's a good point. We probably should, I'll add it to the next PR!